lib/roo_on_rails/railties/database.rb

```diff
@@ -0,0 +1,17 @@
+module RooOnRails
+  module Railties
+    class Database < Rails::Railtie
+      if ActiveRecord::VERSION::MAJOR >= 4
```

Comment:
I'm not sure this is really needed; setting the variables doesn't _hurt_, it just doesn't do anything.